gio/Gir.toml

```diff
@@ -1168,13 +1241,32 @@ name = "Gio.Vfs"
 status = "generate"
 concurrency = "send+sync"
 
+[[object]]
+name = "Gio.Volume"
+status = "generate"
+    [[object.function]]
+    name = "get_icon"
+        [object.function.return]
+        nullable = false
```

Comment:
Why all the `nullable = false` here?

Reply:
I can't tell now... and it doesn't seem to make any difference. Removing.